app/logic/Calendar/Event.ts

```diff
@@ -250,6 +249,20 @@ export class Event extends Observable {
     return !this.dbID;
   }
 
+  get seriesStatus() {
```

Comment:
Please define the return type, and document all values.
Please add a description comment of what this tells us.